### DISCUSSION TITLE
There is only one China

### DISCUSSION THREAD

**Discussion by LiZheGuang** [strapi/strapi#6503] [Legacy Topics]
There is only one China！

You can distinguish between traditional and simplified characters！
Does the national flag of the kuomintang represent traditional Chinese characters?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
